util/reggen/reg_pkg.sv.tpl

```diff
@@ -169,8 +169,14 @@ packbit = 0
     msb = nbits - packbit - 1
     lsb = msb - struct_width + 1
     packbit += struct_width
+    string = "{} {}; // [{}:{}]".format(struct_type, r0.name.lower(), msb, lsb)
```

Comment:
Could this have a slightly more descriptive name? Maybe `var_decl` or something? But, better, I think you can avoid repeating yourself by refactoring slightly: (untested!)
```
    name_and_comment = f'{r0.name.lower()}; // [{msb}:{lsb}]'
%>\
  % if 4 + len(struct_type) + 1 + len(name_and_comment) <= 100:
    ${struct_type} ${name_and_comment}
  % else:
    ${struct_type}
      ${name_and_comment}
  % endif
```
And obviously the same for the other hunk.

Reply:
Yes this looks cleaner - changed!